.github/CODEOWNERS

```diff
@@ -933,6 +930,12 @@
 # ServiceLabel: %Mgmt
 # AzureSdkOwners:                                                  @ArthurMa1978
 
+# ServiceLabel: %Network - CDN %Mgmt
```

Comment:
Move to after the PR section, please, so that this is consistent with the other entries in the file.